apps/previewgenerator.sh

```diff
@@ -16,18 +16,32 @@ debug_mode
 # Check if root
 root_check
 
-msg_box "Please note that this will put your server at risk as it will install a package called 'imagick' which is known to have several flaws.
+msg_box "This script will install the previewgerator. 
 
-You can check this issue to understand why: https://github.com/nextcloud/vm/issues/743
-
-You can choose to cancel installing this in the next step."
-if [[ "no" == $(ask_yes_or_no "Do you still want to continue?") ]]
+It can speedup the feel of Nextcloud by a lot."
+if [[ "no" == $(ask_yes_or_no "So do you want to install the previewgenerator?") ]]
 then
     exit
 else
     sleep 1
 fi
 
+msg_box "In the next step you can choose to install a package called imagick to speed up the generation of previews and get support for many filetypes. 
```

Comment:
I'm not sure but doesn't the previewgenerator depend on imagick? In other words, if not installed previewgenerator will not function?

Reply:
No, I am currently running the preview:generate-all command on my instance without imagick installed and it works nicely.